FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -253,6 +253,38 @@ def do_exit(self, *ignored):
         completer=lambda self: self.fab.getTiles(),
     )
 
+    install_oss_cad_suite_parser = Cmd2ArgumentParser()
+    install_oss_cad_suite_parser.add_argument(
+        "destination_folder",
+        type=Path,
+        help="Destination folder for the installation",
+        default="",
+        completer=Cmd.path_complete,
+        nargs=argparse.OPTIONAL,
+    )
+    install_oss_cad_suite_parser.add_argument(
+        "update_existing",
```

Comment:
I think using `--update` is more natural. Can use `action="store_true"` which make it a flag option.

Reply:
Good idea, I've implemented it now similar to the create project flag and made both flags mutual exclusive  :)